src/main/java/com/github/thundermarket/thundermarket/service/ProductService.java

```diff
@@ -0,0 +1,34 @@
+package com.github.thundermarket.thundermarket.service;
+
+import com.github.thundermarket.thundermarket.domain.Product;
+import com.github.thundermarket.thundermarket.repository.ProductRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class ProductService {
+
+    private final ProductRepository productRepository;
+
+    public ProductService(ProductRepository userRepository) {
+        this.productRepository = userRepository;
+    }
+
+
+    public List<Product> getProductList() {
```

Comment:
List라는 표현은 함수, 메소드, 변수 이름에 되도록이면 붙이지 말아주세요. 리스트 뿐만 아니라 특정 자료형을 표현하는 이름은 되도록이면 삼가해주세요. 자료형은 언제든지 바뀔 수 있어서요. 자료형을 표현한게 아니라고 하더라도 오해의 여지가 있어요. 아래처럼 제안 드려요. (optional; 그리고 이건 취향인데요. 요즘에는 get이라는 표현 없이 그냥 반환되는 모델을 그대로 작성해요.)
```suggestion
    public List<Product> products() {
```

Reply:
686b6a2087f0ea92e12c54a4fc4fc472a86f2ed3
메서드 명을 `products` 로 변경했습니다.
헝가리안 표기법 처럼 자료형이 들어가는 네이밍을 가끔 .. 홀린듯이 쓸 때가 있는 것 같아요.
유의해서 지어보겠습니다.

get 안 붙이는 방법이 있는건 처음알았네요! 깔끔한 것 같아요 감사합니다